composer.json

```diff
@@ -50,19 +51,27 @@
     "require-dev": {
         "phpcr/phpcr-shell": "~1.0",
         "jackalope/jackalope-doctrine-dbal": "^1.2.5",
-        "guzzle/plugin-mock": "3.9.*",
         "massive/build-bundle": "dev-master",
         "matthiasnoback/symfony-config-test": "^1.1",
         "matthiasnoback/symfony-dependency-injection-test": "^0.7.3",
         "phpunit/phpunit": "^4.8.31",
         "phpspec/prophecy": "^1.4",
-        "symfony/phpunit-bridge": "~2.8.7, !=2.8.10",
+        "symfony/phpunit-bridge": "^2.8.7 || ^3.0",
         "sensio/framework-extra-bundle": "~3.0",
-        "symfony/monolog-bundle": "~2.8 || ~3.0",
+        "symfony/monolog-bundle": "^2.8.7 || ^3.0",
+        "twig/twig": "~1.11",
         "zendframework/zend-stdlib": "~2.3",
         "zendframework/zendsearch": "@dev",
-        "doctrine/doctrine-fixtures-bundle": "~2.2"
+        "doctrine/doctrine-fixtures-bundle": "~2.3",
+        "phpdocumentor/reflection-docblock": "~3.1"
+    },
+    "conflicts": {
+        "symfony/symfony": "2.8.10",
+        "symfony/phpunit-bridge": "2.8.10",
+        "symfony/security-acl": "2.8.10",
+        "symfony/monolog-bundle": "2.8.10"
     },
+    "minimum-stability": "RC",
```

Comment:
no i would merge it with that. the developer can decide in the real application if he wants that.